.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
`.pytest_cache` should be removed (commit) and then added to `.gitignore`.

Reply:
done